core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -10268,6 +10270,17 @@ protected UniqueIDPaginatedSearchResult doGetUserListWithID(Condition condition,
         throw new NotImplementedException("doGetUserListWithID operation is not implemented in: " + this.getClass());
     }
 
+    //TODO - If the LDAP method also implements cursor like the JDBC one, then the above method can be removed
+    // (Edit: it cannot be removed)
+    protected UniqueIDPaginatedSearchResult doGetUserListWithID(Condition condition, String profileName, int limit,
```

Comment:
Method cannot be merged with the above method as it is called by several other methods which are not used which my code